truffleruby/README.md

```diff
@@ -7,28 +7,38 @@ TruffleRuby is the [GraalVM](http://graalvm.org/) high-performance implementatio
 
 See https://github.com/graalvm/container/pkgs/container/truffleruby
 
-## How to use these images
+## Supported tags
 
-The images are intended for use in the **FROM** field of a downstream Dockerfile. For example, specify `FROM ghcr.io/graalvm/truffleruby:latest` or a version tag.
+* Oracle Linux 8: [`latest`/`VERSION`](https://github.com/graalvm/container/blob/master/truffleruby/Dockerfile.ol8).  
```

Comment:
This is also Oracle Linux 8 slim, the difference is in the llvm-toolchain availability.

Reply:
Right, will clarify

Reply:
done